projects/prison-custody-status-to-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/integrations/delius/person/manager/prison/PrisonManagerService.kt

```diff
@@ -54,8 +58,20 @@ class PrisonManagerService(
     ) {
         val person = disposal.event.person
 
+        val activePrisonManager: PrisonManager?
+        try {
+            activePrisonManager = prisonManagerRepository.findActiveManagerAtDate(person.id, allocationDate)
+        } catch (e: IncorrectResultSizeDataAccessException) {
+            if (personRepository.findByMergedFromCrn(person.id) == null) {
+                throw e
+            }
+            //where crn has a merged from record
+            //we can ignore the IncorrectResultSizeDataAccessException and
+            //process the record no further
+            throw IgnorableMessageException("Person has merged from record", mapOf("person id" to person.id.toString()))
```

Comment:
I think the NOMS number is automatically added to these - so no need to add the person id manually.  Also, the other IgnorableMessageException instances in this service seem to have a slightly different message format - probably worth making this one consistent e.g. something like
```suggestion
            throw IgnorableMessageException("MergedPrisonManagerHistory")
```

Reply:
already added

Reply:
What I mean is you don't need to add any additional properties here, the NOMS number is already added by default in the Handler: https://github.com/ministryofjustice/hmpps-probation-integration-services/blob/559f1b6a261fc25883c0ed70f1e03919f7b22df5/projects/prison-custody-status-to-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/messaging/Handler.kt#L110-L115

Also could we shorten the exception message to be consistent with the others in this project?  e.g. 
```
throw IgnorableMessageException("MergedPrisonManagerHistory")
```

Reply:
the action processor allow additional properties.  how else will will tie the message with the crn if we do not add the identifier.  i am happy with the identifier, and would need further convincing to change

Reply:
Ah! Sorry, not trying to be a pain honestly, but I just had a look through the existing code and realised we can't use an IgnorableMessageException here anyway 🤦 as it will interrupt the UpdateLocationAction and means it won't create the location change contact.  So we need to change this to just `return` anyway!

Reply:
would we not already have created a updated contact from the pre merge message.  why do we need to create a contact for the location change?